TrainingManager.py

```diff
@@ -48,7 +42,7 @@ def __init__(self):
         
 
     
-    def training_step(self, train_loader):
+    def training_step(self, train_loader) -> float:
```

Comment:
I meant the `train_loader` argument should be statically typed:

`def training_step(self, train_loader: DataLoader) -> float:`